src/lib/platform/OSXKeyState.cpp

```diff
@@ -108,22 +97,21 @@ static const KeyEntry    s_controlKeys[] = {
     // to map to.  also the enter key with numlock on is a modifier but i
     // don't know which.
 
-    // modifier keys.  OS X doesn't seem to support right handed versions
-    // of modifier keys so we map them to the left handed versions.
-    { kKeyShift_L,        s_shiftVK },
-    { kKeyShift_R,        s_shiftVK }, // 60
-    { kKeyControl_L,    s_controlVK },
-    { kKeyControl_R,    s_controlVK }, // 62
-    { kKeyAlt_L,        s_altVK },
-    { kKeyAlt_R,        s_altVK },
-    { kKeySuper_L,        s_superVK },
-    { kKeySuper_R,        s_superVK }, // 61
-    { kKeyMeta_L,        s_superVK },
-    { kKeyMeta_R,        s_superVK }, // 61
+    // modifier keys. Support for right side keys is a later addition in macOS.
+    { kKeyShift_L,          kVK_Shift },        // 56
```

Comment:
Let's align with the comma (`{ kKeyShift_L, kVK_Shift },`), because otherwise all lines will need to be modified whenever a very long first work appears in the list.

Reply:
fixed